instruqt/avoiding-installation-pitfalls/02-checking-cluster-resources/assignment.md

```diff
@@ -143,24 +143,18 @@ Now that we have a thorough set of preflights for cluster resources, let's run
 them:
 
 ```
-kubectl preflight ./harbor-preflights.yaml
+kubectl preflight ./slackernews-preflights.yaml
 ```
 
 You'll see that all three preflights are run, and that the memory
 preflight has failed. This is an expected failure, since we have
-single node cluster that uses just that node's disk for storage. That
-disk is smaller than the storage requirements for Harbor.
+single node cluster that has a small memory footprint.
 
 ![Failing Storage Preflight](../assets/memory-preflight-failure.png)
 
 If you completed the [Distributing Your Application with Replicated](https://play.instruqt.com/replicated/tracks/distributing-your-application-with-replicated)
 lab, you may be surprised at this failure. In that lab, we deployed
-Harbor and it came up successfully. This is another value provided by
+Slackernews and it came up successfully. This is another value provided by
```

Comment:
> This is another value provided by your checks. 

I read this as value meaning like the value of a variable, coming out of the code context that comes right before it, but I think this should read  "This is another benefit to using preflight checks..."

Reply:
Dug into this one and the previous one a little bit more and finally got it. Reworked it for clarity.